format.go

```diff
@@ -16,7 +13,7 @@ const (
 type Units uint
 
 const (
-	UnitNone Units = iota
```

Comment:
If you think this is problematic because there may be other people already using it, I can remove the change.
Otherwise, I'd recommend removing it the sooner the better.